src/browser_launchers/playwright_launcher.js

```diff
@@ -36,20 +36,24 @@ export class PlaywrightLauncher extends BrowserLauncher {
 
     /**
     * @param {PlaywrightLaunchContext} launchContext
-    * All `PlaywrightLauncher` parameters are passed via an launchContext object.
+    * All `PlaywrightLauncher` parameters are passed via this launchContext object.
     */
     constructor(launchContext = {}) {
         ow(launchContext, 'PlaywrightLauncherOptions', ow.object.exactShape(PlaywrightLauncher.optionsShape));
 
-        launchContext.launcher = launchContext.launcher || require('playwright').chromium; // eslint-disable-line
+        const {
+            launcher = BrowserLauncher.requireLauncherOrThrow('playwright').chromium,
+        } = launchContext;
+
+        launchContext.launcher = launcher;
```

Comment:
Ok, I see why you did not use the default params before. I'd probably do it like below, because it's not polite to modify the object you're receiving.

```js
const {
    launcher = BrowserLauncher.requireLauncherOrThrow('playwright').chromium,
} = launchContext;

super({
    ...launchContext,
    launcher,
})
```